src/content/reference/react/use.md

```diff
@@ -0,0 +1,769 @@
+---
+title: use
+---
+
+<Wip>
+This section is incomplete. See also the [draft use RFC](https://github.com/acdlite/rfcs/blob/first-class-promises/text/0000-first-class-support-for-promises.md#usepromise).
+</Wip>
+
+[//]: # (TODO: replace the `<Note></Note>` below with `<Canary></Canary>` once https://github.com/reactjs/react.dev/pull/6173 is merged)
+<Note>
+The `use` hook is only supported when using [React Server Components](/learn/start-a-new-react-project#bleeding-edge-react-frameworks). React Server Components are only available in React’s canary and experimental channels. Learn more about [React's release channels here](/community/versioning-policy#all-release-channels). 
+</Note>
+
+<Intro>
+
+`use` is a React Hook that lets you read the value of a resource like a [promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) or [context](/learn/passing-data-deeply-with-context).
+
+```js
+const value = use(resource);
+```
+
+</Intro>
+
+<InlineToc />
+
+---
+
+## Reference {/*reference*/}
+
+### `use(resource)` {/*use*/}
+
+Call `use` in your component to read the value of a resource like a [promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) or [context](/learn/passing-data-deeply-with-context).
+
+```jsx
+import { use } from "react";
+
+function Message({ messagePromise }) {
+  const message = use(messagePromise);
+  const theme = use(ThemeContext);
+  //...
+```
+
+Unlike all other React Hooks, the `use` Hook can be called within conditional statements like loops and `if` statements. Like other React Hooks, the parent function of a Hook must be a React component or Hook.
```

Comment:
nit: "parent function" here and below is a bit jargony; do you think "must be called from" or "called within" suffices? or is that confusing because it sounds like it could be a called from an intermediate helper

Reply:
Good point, I've updated this to "Like other React Hooks, the function that calls `use` must be React component or Hook." and updated the rest of this page's references to "parent function" to be less jargony. What do you think?